packages/react-instantsearch-hooks/src/usePoweredBy.ts

```diff
@@ -0,0 +1,12 @@
+import { safelyRunOnBrowser } from 'instantsearch.js/es/lib/utils';
+
+import type { PoweredByRenderState } from 'instantsearch.js/es/connectors/powered-by/connectPoweredBy';
+
+export function usePoweredBy(): PoweredByRenderState {
+  return {
+    url: `https://www.algolia.com/?utm_source=react-instantsearch&utm_medium=website&utm_content=${safelyRunOnBrowser(
```

Comment:
Can we extract the `safelyRunOnBrowser()` call to a variable to ease reading?

Reply:
Addressed in c1ef574.